packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -77,13 +77,17 @@ function Navbar() {
   );
 
   const logoLink = logo.href || baseUrl;
-  const isExternalLogoLink = /http/.test(logoLink);
-  const logoLinkProps = isExternalLogoLink
-    ? {
-        rel: 'noopener noreferrer',
-        target: '_blank',
-      }
-    : null;
+  let logoLinkProps = {};
+
+  if (logo.target) {
+    logoLinkProps = {target: logo.target};
+  } else if (/http/.test(logoLink)) {
```

Comment:
We have an `@docusaurus/isInternalUrl()` to use here.

Reply:
Done! :heavy_check_mark: